src/components/base/Sitenav.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div v-if="showNavbar" class="sitenav">
+    <div class="sidenav-content">
+      <h4 class="sidenav-headline">Rooms-Admin</h4>
+      <hr/>
+      <div class="sidenav-links" v-for="links in link" :key="links.id">
+        <sitenavLink :title="links.title" :link="links.link" :icon="links.icon"></sitenavLink>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import sitenavLink from '@/components/sitenav/sitenav-link'
+export default {
+  name: 'sitenav',
+  data: function () {
```

Comment:
In der Data bitte nicht von EN auf DE wechseln.

Reply:
Done